php/php.editor/test/unit/src/org/netbeans/modules/php/editor/PHPBracesMatcherTest.java

```diff
@@ -522,7 +522,7 @@ public void testAttributeSyntax_05b() throws Exception {
                 + "    #[Class1(5)]\n"
                 + "    public function method~(#[Class1(6)] $param1, #[Class1('foo', 'bar', 7)] int $pram2@)^ {}\n"
                 + "\n"
-                + "@}^"
+                + "}"
```

Comment:
Why these changes exactly?

Reply:
Maybe, I forgot to remove them when I copied and pasted the previous test.
`@^~` are used to get positions. (e.g. `^` is caret position)
1. get positions using the `indexOf()` method
2. replace all `@^~` to empty string (`""`)

So, the extra markers did not affect unit tests.

Reply:
I see, thanks for the explanation.